go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.10.0
 	github.com/wI2L/jsondiff v0.6.0
-	go.goms.io/fleet-networking v0.3.5
+	go.goms.io/fleet-networking v0.3.3
```

Comment:
why revert this?

Reply:
i reverted two commits (including these) and we can upgrade the networking package later if needed .